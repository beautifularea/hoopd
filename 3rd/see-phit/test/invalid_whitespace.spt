R"*(
< HTML>
  <DIV> 
    Space after open angle bracket
  </DIV>
</HTML>  
)*"_html; 