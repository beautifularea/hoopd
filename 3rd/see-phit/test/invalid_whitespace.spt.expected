Error() [with int ROW = 2; int COL = 2; WHAT = spt::Expecting_a_tag_name_after_open_bracket]':
